demos/storybook/stories/selection-toolbar/with-nav-icon.stories.ts

```diff
@@ -0,0 +1,30 @@
+import { text } from '@storybook/addon-knobs';
+
+export const withNavIcon = (): any => ({
+    template: `
+       <pxb-selection-toolbar [title]="title" [subtitle]="state.selected || subtitle">
+            <button mat-icon-button pxb-icon (click)="clickPXBIcon()" aria-label="menu icon">
+                <mat-icon>menu</mat-icon>
```

Comment:
Can we throw a knob in here for which icon to use (maybe menu versus back arrow)? Otherwise the knobs are the same as basic usage.